proteus/proteus/tests/test_action.py

```diff
@@ -0,0 +1,49 @@
+# This file is part of Tryton.  The COPYRIGHT file at the top level of
+# this repository contains the full copyright notices and license terms.
+
+import unittest
+
+try:
+    import pydot
+except ImportError:
+    pydot = None
+
+from proteus import Model, Wizard, launch_action
+
+from .common import ProteusTestCase
+
+
+class TestAction(ProteusTestCase):
+
+    def test_act_window(self):
+        IrModel = Model.get('ir.model')
+        actions = IrModel.find([
+                ('name', '=', 'ir.action'),
+                ])
+        model_access = launch_action(
+            'ir.act_model_access_form_relate_model', actions)
```

Comment:
Using `actions` as a variable name is a little misleading, it should probably be records (I needed some time to understand properly)